common/src/main/java/com/sequenceiq/cloudbreak/auth/altus/Crn.java

```diff
@@ -144,7 +144,8 @@ public static Partition safeFromString(String input) {
         DSADMIN("dsadmin", ADMIN_SERVICE),
         SAMPLE("sample", NON_ADMIN_SERVICE),
         WORKSPACES("workspaces", NON_ADMIN_SERVICE),
-        FREEIPA("freeipa", NON_ADMIN_SERVICE);
+        FREEIPA("freeipa", NON_ADMIN_SERVICE),
```

Comment:
where we using this ?

Reply:
In [CrnService.java](https://github.com/hortonworks/cloudbreak/blob/681e20a6f15ac4b5a648892df9ff0f73bc1870a0/freeipa/src/main/java/com/sequenceiq/freeipa/util/CrnService.java#L66)

Reply:
It seems we are using it here: https://github.com/hortonworks/cloudbreak/blob/681e20a6f15ac4b5a648892df9ff0f73bc1870a0/freeipa/src/main/java/com/sequenceiq/freeipa/service/stack/FreeIpaCreationService.java#L102